broker/src/test/java/io/zeebe/broker/logstreams/AtomixLogDeletionServiceTest.java

```diff
@@ -147,7 +147,7 @@ private void createSnapshot(final long index) {
     try {
       return builder
           // hardcode max segment size to allow a single entry only
-          .withMaxSegmentSize(JournalSegmentDescriptor.BYTES + 8 * Integer.BYTES)
+          .withMaxSegmentSize(JournalSegmentDescriptor.BYTES + 9 * Integer.BYTES)
```

Comment:
Why is this required?

Reply:
That was necessary in the 0.25 changes because the serialized entry was bigger with the new serializer but here we're still using the old one so it shouldn't be necessary. I just forgot to remove it